template.tpl

```diff
@@ -1248,10 +1255,11 @@ function callSDKForEvent(eventData) {
           }
         }
       }
+      var mergedIdentities = mergeUserPropsAndIdentities(userProperties,identities);
+      log('identities are ...', identities);
+      log('setUserIdentities getting invoked with identities merged with user props... : ', mergedIdentities);
       
-      log('setUserIdentities getting invoked with ... : ', identities);
-      
-      callInWindow('zeotap.callMethod', 'setUserIdentities', identities, data.areIdentitiesHashed);
+      callInWindow('zeotap.callMethod', 'setUserIdentities', mergedIdentities, data.areIdentitiesHashed);
```

Comment:
@rishabh-zeo should we add user props in all api calls instead of only setUserIdentities call?

Reply:
once in setUserIdentities, properties will be persisted for all calls right. from identity cookie